VTIL-Architecture/CMakeLists.txt

```diff
@@ -12,3 +12,11 @@ source_group(TREE ${PROJECT_SOURCE_DIR} FILES ${SOURCES} ${INCLUDES})
 
 target_include_directories(${PROJECT_NAME} PUBLIC includes)
 target_link_libraries(${PROJECT_NAME} VTIL-Common VTIL-SymEx)
+
+configure_file(VTIL-Architecture.pc.in ${CMAKE_BINARY_DIR}/VTIL-Architecture.pc @ONLY)
+
+if(VTIL_INSTALL)
+    install(TARGETS ${PROJECT_NAME} DESTINATION lib)

```

Comment:
This doesn't actually create a `lib/cmake` folder with the correct CMake targets. Do you know how to set it up in that way (so you can use `find_package(VTIL 0.1.0 REQUIRED)` and use the `VTIL-Common`, `VTIL-SymEx` etc? That way you can install and use a VTIL package on Windows (and not just with this .pc.in pkg-config crap)

Reply:
> This doesn't actually create a lib/cmake folder with the correct CMake targets

Yes, I have not added CMake support.

> That way you can install and use a VTIL package on Windows (and not just with this .pc.in pkg-config crap).

Currently working on packaging for Linux and macOS, as CMake can integrate with pkg-config. I plan on getting to CMake later.

> Additionally this doesn't install capstone/keystone which are also required.

I need to figure out how to handle packaging private headers, then I will decide how to handle Capstone and Keystone.